src/Misc/fileTools.ts

```diff
@@ -87,7 +87,11 @@ export class FileTools {
      * Gets or sets a function used to pre-process url before using them to load assets
      */
     public static PreprocessUrl = (url: string) => {
-        return url;
+        let processedURL = url.substring(5).toLowerCase();

```

Comment:
This function is actually for the external users and used during the requestFile processing. Meaning that if baseUrl is set to "" then we will strip off the moniker and it will not work anymore if the content is not a file

I recommend to do it only when url.indexOf("file) is > -1 (inside the loadFile function)

Reply:
reverted the change